config.ru

```diff
@@ -1,5 +1,3 @@
-# frozen_string_literal:true
```

Comment:
Let's leave this in the file, we don't have rubocop to enforce style rules yet, but these should be included in all ruby code